libs/regulations/src/lib/types-web.ts

```diff
@@ -266,3 +267,48 @@ export enum RegulationOriginalDates {
   gqlHack = '0101-01-01',
   api = 'original',
 }
+
+// ---------------------------------------------------------------------------
+
+/** Input data for regulation PDF generation */
+export type RegulationPdfInput = {
```

Comment:
These types are part of the regulations-admin application, and should therefore be placed in the file `./types-admin.ts`

This file, `types-web.ts` is for more generic, public facing types in the `web` project - some of which may be consumed by the admin project also, but admin-only types should preferably be kept in `types-admin.ts` and imported from `@island.is/regulations/admin`

Reply:
**N.B.** there will be a few exclusively `admin`-related API endpoints/methods that we'll need to generate (file-uploads, more PDF generation, etc.) so it's not just the matter of this single server method.